pkgs/applications/blockchains/lnd/default.nix

```diff
@@ -19,17 +20,7 @@ buildGoModule rec {
 
   subPackages = [ "cmd/lncli" "cmd/lnd" ];
 
-  preBuild = let
-    buildVars = {
-      RawTags = lib.concatStringsSep "," tags;
-      GoVersion = "$(go version | egrep -o 'go[0-9]+[.][^ ]*')";
```

Comment:
These ldflags are not required, upstream read these values from runtime, see https://github.com/lightningnetwork/lnd/blob/2fb150c8fe827df9df0520ef9916b3afb7b03a8d/build/version.go#L66-L75